### ISSUE TITLE
Please confirm expected output for remote harvester.

### ISSUE THREAD

**Issue by dxyzab** [Chia-Network/chia-blockchain#3157]
**Describe the issue**
After following the instructions at https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines
and confirming the connections in logs are as expected. On the harvester 

chia farm summary shows 

Connection error. Check if wallet is running at 9256
Connection error. Check if full node is running at 8555
Connection error. Check if farmer is running at 8559
Farming status: Not available
Total chia farmed: Unknown
User transaction fees: Unknown
Block rewards: Unknown
Last height farmed: Unknown
Plot count: 91
Total size of plots: 9.008 TiB
Estimated network space: Unknown
Expected time to win: Unknown

**To Reproduce**
Set up a remote harvester according to the instructions.

**Expected behavior**
I would expect to see a connection to the wallet, full node and the farmer on the main machine.

**Screenshots**

**Additional context**
This is concerning as it suggests any rewards on the remote harvester will not be put into the wallet.

**Comment by kswtch** [comment]
On your Harvester and Main Node change your Logging output to "INFO" and check ~/.chia/mainnet/log/debug.log. 

On Harvesters you should see:
```
2021-04-27T20:09:37.560 harvester harvester_server        : INFO     <- new_signage_point_harvester from peer af06d005f0d2xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxfc25 <ip_main_node>
2021-04-27T20:09:37.561 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming 6a5678519d... Found 0 proofs. Time: 0.00130 s. Total <num_plots> plots
2021-04-27T20:09:37.562 harvester harvester_server        : INFO     -> farming_info to peer <ip_main_node> af06d005f0d2xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxfc25
```
On Main Node you should see:
```
2021-04-30T14:54:50.729 farmer farmer_server              : INFO     -> new_signage_point_harvester to peer <harvester_ip 893c6xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx61aa
2021-04-30T14:54:50.758 farmer farmer_server              : INFO     <- farming_info from peer 893c6xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx61aa <harvester_ip>

```

**Comment by dxyzab** [comment]
These were check and are present.

My concern is that the harvester will not be able to connect to the wallet should it find a proof. How can I verify this?

**Comment by kswtch** [comment]
From my understanding:
Your harvesters don't need to connect to the wallet. In fact they can't. They connect to the farmer. The farmer connects to the Full Node. The wallet also connects to the Full Node. See this image as an example: https://raw.githubusercontent.com/wiki/Chia-Network/chia-blockchain/images/chia-network-architecture.png

**Comment by dxyzab** [comment]
I see - thank you for the diagram. Its just a little worrying to the novice (me) that I won't get the rewards should my harvester find a proof. I seems my harvester is connected to the farmer.

If the main node were able to report the status of the entire farm this would be helpful. I upvoted
https://github.com/Chia-Network/chia-blockchain/issues/3194

Thanks Daniel for your help.

**Comment by franco-101** [comment]
Does this mean I need to keep the full node running? Can I just not have the harvester running separately on a RPI with no full node?

**Comment by thauburger** [comment]
There always needs to be at least one full node (synced) somewhere in your system to farm successfully.

It does not have to be on the same machine (necessarily) as the harvester, so long as the remote harvester is configured to talk to a farmer service that does have access to that full node.

Example (with fake IP addresses for illustration)

_Primary_
**Machine A (10.0.0.1)**: Runs full node, farmer, wallet, and harvester service surveying plots on A's storage

_Harvesters_
**Machine B (10.0.0.2)**: Remote harvester service only - surveying plots on B's storage
**Machine C (10.0.0.3)**: Remote harvester service only - surveying plots on C's storage

B and C do NOT need to host a full node (or farmer, or wallet), so long as you follow the configuration steps for [Farming on many machines](https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines)

Basically, on each pure harvester node (B and C), you just need to follow the cert instructions (using's A's certs for signing), and point B and C each toward A as a farmer peer.

The most challenging part of this process is that it's hard to see your total active farm size (and estimated time to win) because Chia doesn't do a great job of consolidating plot and farming information across multiple harvesters. So you have to do a sanity check in logs.

There's an open feature request for better remote harvester support in #3194

**Comment by gd1230** [comment]
In some cases, smaller setup, would it just be easier to plot on an offline device or 3, and move them to the main farmer that stays online? Seems more manual work, but better piece of mind?

**Comment by gd1230** [comment]
Is there an advantage to spreading out plots over a few harvesters as opposed to one main machine? guess I am missing the point of the harvester setup then if there is no success improvement.

**Comment by kswtch** [comment]
You are probably only interested in a setup like this if you
- have your storage spread out over multiple locations.
- have attached so many storage units to a regular PC that your time to check for and report proofs exceeds 30 seconds.

**Comment by franco-101** [comment]
How can you test the proof speed? What sort of size of storage are you talking about?
